src/common.jl

```diff
@@ -84,10 +97,10 @@ function frequencies(input::AbstractString)
     ans = Dict{Char, Int}()
     for c in input
         index = Base.ht_keyindex2!(ans, c)
```

Comment:
This function is probably now better expressed as "map the function `(+) 1` over the values of `_frequencies_internal(input)`.

Reply:
In fact it may be better to just have the one `_frequencies` function, but which does correctly count from `1`, and then expose it publicly and tweak any usages of it to subtract 1 if needed.